Python/bytecodes.c

```diff
@@ -1542,7 +1542,7 @@ dummy_func(
             (void)counter;
         }
 
-        op(_UNPACK_SEQUENCE, (seq -- unused[oparg], top[0])) {
```

Comment:
Would it be possible to tell the validator that `unused` matches anything (both ways)?

So `unused` in optimizer_bytecodes/bytecodes isn't name-validated.